MotionPlanningEnv/sphereObstacle.py

```diff
@@ -1,73 +1,66 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
+import csv
+from typing import List, Optional
 import numpy as np
+
 import os
-import csv
+from omegaconf import OmegaConf
+
+from MotionPlanningSceneHelpers.motionPlanningComponent import (
+    DimensionNotSuitableForEnv,
+)
 from MotionPlanningEnv.collisionObstacle import (
     CollisionObstacle,
     CollisionObstacleConfig,
 )
-from MotionPlanningSceneHelpers.motionPlanningComponent import (
-    DimensionNotSuitableForEnv,
-)
-
-from omegaconf import OmegaConf
-from typing import List, Optional
-
-
-class SphereObstacleMissmatchDimensionError(Exception):
-    pass
-
 
 @dataclass
 class GeometryConfig:
-    """Configuration dataclass for geometry.
-
-    This configuration class holds information about position
-    and radius of a sphere obstacle.
-
+    """
+    This configuration class holds information about
+    the geometry of the sphere obstacle.
     Parameters:
     ------------
-
-    position: list: Position of the obstacle
     radius: float: Radius of the obstacle
     """
-
-    position: List[float]
     radius: float
 
-
 @dataclass
 class SphereObstacleConfig(CollisionObstacleConfig):
-    """Configuration dataclass for sphere obstacle.
-
-    This configuration class holds information about the position, size
-    and randomization of a spherical obstacle.
-
+    """
+    Configuration dataclass for sphere obstacle.
     Parameters:
     ------------
-
     geometry : GeometryConfig : Geometry of the obstacle
+    orientation: list: Orientation of the obstacle
```

Comment:
I want to put orientation, movable, mass, color in the ColissionObstacle class. But CollisionObstacle cannot have default
values because SphereObstacle has non-default values. Resulting in:
```__init__(self, param1, param2, param3 = 15, param4)```
See param3 has a default value, while param4 has not.